proposals/lip-0037.md

```diff
@@ -33,23 +33,25 @@ Replay attacks for transactions and blocks, i.e. replaying transactions or block
 
 ### Message Tags
 
-Prepending a tag to a binary message before signing, where the tag is specific to the type of the message, prevents the signature re-usage in general. Consider again the blockheader `bh` and the transaction `tx` that are serialized to the same binary message `bm`. When the tag `"LSK_BH_"` is used for block header signatures and the tag `"LSK_TX_"` for transaction signatures, then the signatures for `bh` and `tx` are computed by signing the messages `"LSK_BH_" || bm` and `"LSK_TX_" || bm` respectively, where `"||"` denotes concatenation. Hence, the signature of `bh` cannot be a valid signature for `tx` and vice versa (assuming no collisions in the signing function).
+Prepending a tag to a binary message before signing, where the tag is specific to the type of the message, prevents the signature re-usage in general. Consider again the blockheader `bh` and the transaction `tx` that are serialized to the same binary message `bm`. When the tag `"LSK_BH_"` is used for block header signatures and the tag `"LSK_TX_"` for transaction signatures, then the signatures for `bh` and `tx` are computed by signing the messages `"LSK_BH_" + bm` and `"LSK_TX_" + bm` respectively. Hence, the signature of `bh` cannot be a valid signature for `tx` and vice versa (assuming no collisions in the signing function).
 
 Note that any updates to the serialization specification of a message type must be accompanied by introducing a new tag. For example, when the [transaction schema](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0028.md#transactionschema-schema) is updated, a new tag like `"LSK_TX:V2_"` must be defined and used.
 
-### Network Identifiers
+### Chain Identifiers
 
-Network identifiers for transaction signatures and block signatures were already introduced in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#update-to-the-block-header-signing-procedure) respectively to prevent replay attacks on other chains. Here, we specify how to use a network identifier for any kind of message to prevent replay attacks on other chains in general.
+Chain identifiers for transaction signatures and block signatures were already introduced in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#update-to-the-block-header-signing-procedure) (as network identifiers) to prevent replay attacks on other chains. Here, we specify how to use a chain identifier for any kind of message to prevent replay attacks on other chains in general.
```

Comment:
I would write here explicitly that the specifications in this LIP supersede those in LIP 9. The way in which we use chain ID for signing is the same as we used network identfiers, however, the length and how they are obtained is now different. I think it would be nice to also elaborate on the reasoning, for instance, something like the following:

> For users to be able to easily verify that they sign a transaction for the correct blockchain, we choose to abandon the usage of 32-byte network identifiers that are computed as a hash value from the genesis block ID and community identifier. Instead, the new chain identifier which also serve the purpose to prevent transaction replay among different blockchains, has a length of only 4 bytes. It further can be chosen by the blockchain creator. For the official mainnet blockchain of a project, we always recommend to register the identifier on the Lisk Mainchain to ensure uniqueness in the Lisk ecosystem.

Reply:
added something in this direction

Reply:
I like the new formulation :-)